modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -211,7 +211,16 @@ export class Wallet implements IWallet {
    * This is a strict sub-set of prebuildWhitelistedParams
    */
   prebuildConsolidateAccountParams(): string[] {
-    return ['consolidateAddresses', 'feeRate', 'maxFeeRate', 'memo', 'validFromBlock', 'validToBlock', 'preview'];
+    return [
+      'consolidateAddresses',
+      'feeRate',
+      'maxFeeRate',
+      'memo',
+      'validFromBlock',
+      'validToBlock',
+      'preview',
+      'keepAlive',
+    ];
```

Comment:
Does this need to be in `core` instead of `sdk-coin-dot` implementation? Maybe through inheritance since this is dot only?

Reply:
This is to update the list of whitelisted params when building/sending account consolidations at the wallet level.
Example call via an SDK script:
```
const consolidationTxs = await walletInstance.sendAccountConsolidations({
    walletPassphrase: walletPassphrase,
    keepAlive: false,
  });
```

If there is somewhere I should move this param flag to let me know.

Reply:
Just pushed an update that adds it to the appropriate jsdocs as well